service_broker/service_broker_app.rb

```diff
@@ -119,7 +119,6 @@ def self.app_settings
   end
 
   def github_service
-    github_credentials = self.class.app_settings.fetch("github")
-    GithubServiceHelper.new(github_credentials.fetch("username"), github_credentials.fetch("access_token"))
```

Comment:
Taking out this piece of code is a breaking change for people who have not configured their ENV but are still using a yaml file. To achieve what you intend to do, we recommend use the ENV VAR if it exists, else go back default behavior which is reading from the yaml file.